crawler/helpers/cherrypicker_test_data.py

```diff
@@ -132,7 +132,7 @@ def _create_sample(dt: datetime, index: int, result: str, plate_barcode: str) ->
         plateCoordinate=well_coordinate,
         preferentiallySequence=False,
         mustSequence=False,
-        fitToPick=True,
+        fitToPick=True if result == "positive" else False,
```

Comment:
Just to check, I noticed value in Mongo is capitalised i.e. 'Positive' not 'positive'. Does that matter for the comparison here (and below)? MySQL is case insensitive right?

Reply:
Yup, it is lowercase at this point.  It doesn't really matter what case it is in at this point.  It gets normalised as it gets put into MongoDB here: https://github.com/sanger/crawler/blob/dpl-480-fix-negative-sample-data/crawler/processing/create_plate_exporter.py#L244